### DISCUSSION TITLE
Fine-tuning Whisper Large

### DISCUSSION THREAD

**Discussion by DonDuckDuck** [openai/whisper#1247] [Q&A] [answered]
Hi, guys, I'm trying fine-tune whisper large's performance on mandarian with huggingface transformer. 

Each of my sample is about 6~7mb and I'm fine tuning it with 4 v100s, even though I set batch size to 1, pytorch still throw me oom error.

It also doesn't work when I trying to run it on a single 3090 24GB GPU with 1 batch size.

Any suggestion to get it running:)?

**Comment by jongwook** [comment, marked as answer]
The audio and the labels should be segmented into 30s or shorter chunks, to match the training distribution. I guess it should make the memory usage low enough. Using [mixed precision](https://pytorch.org/docs/stable/amp.html) and [gradient checkpointing](https://pytorch.org/docs/stable/checkpoint.html) may further reduce the memory usage during fine-tuning.
